### PR TITLE
EWPP-3146: Fixing leftover todos from removing TMGMT.

### PR DESCRIPTION
**Pull request by upchuk**

### CODE REVIEW

File: src/TranslationSourceFieldProcessor/DefaultFieldProcessor.php [hunk 1]
```diff
@@ -49,8 +49,6 @@ public static function create(ContainerInterface $container) {
   /**
    * {@inheritdoc}
    *
-   * @todo add logic from https://www.drupal.org/project/tmgmt/issues/2942236
```
Comment (upchuk) [inline src/TranslationSourceFieldProcessor/DefaultFieldProcessor.php:52]: Turns out this was already there.

File: tests/src/Kernel/LinkFieldProcessorTest.php [hunk 2]
```diff
@@ -1,27 +0,0 @@
-<?php
```
Comment (upchuk) [inline tests/src/Kernel/LinkFieldProcessorTest.php:1]: This test was only needed when we were altering tmgmt to use the default field processor.

File: tests/src/Kernel/TranslationSourceTest.php [hunk 3]
```diff
@@ -197,12 +256,18 @@ public function testNodeTranslation(): void {
         'format' => 'unallowed_format',
       ],
     ]);
+    $node->set('ignored_field', 'The ignored field value');
+    // Inform the test even subscriber to add some extra field to the
```
Comment (22Alexandra) [inline tests/src/Kernel/TranslationSourceTest.php:260]:
```suggestion
    // Inform the test event subscriber to add some extra field to the
```

**Review by 22Alexandra — DISMISSED** [review]

**Comment by hernani** [comment]
https://github.com/openeuropa/oe_translation/blob/EWPP-3146-3/oe_translation.install#L35  tmgmt still used

**Comment by hernani** [comment]
https://github.com/openeuropa/oe_translation/blob/EWPP-3146-3/modules/oe_translation_corporate_workflow/oe_translation_corporate_workflow.module#L116

https://github.com/openeuropa/oe_translation/blob/EWPP-3146-3/modules/oe_translation_corporate_workflow/oe_translation_corporate_workflow.module#L287

https://github.com/openeuropa/oe_translation/blob/EWPP-3146-3/modules/oe_translation_corporate_workflow/tests/src/Kernel/CorporateWorkflowTranslationTest.php#L109

https://github.com/openeuropa/oe_translation/blob/EWPP-3146-3/modules/oe_translation_local/tests/src/Functional/LocalTranslationsTest.php#L55

**Comment by hernani** [comment]
Remote translation module needs options module as dependency

**Review by hernani — CHANGES_REQUESTED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**